Update missing-data-methods.md (#4211)
the-turing-way/the-turing-way: book/website/project-design/missing-data/missing-data-methods.md

---
height: 500px
name: missing-data-handling-diagram

> alt: Flowchart about Missing Data Handling Techniques, adapted from Joel et al., 2022. The diagram presents three main strategies: Deletion, Imputation, and Built-in. Under Deletion, two methods are listed: Listwise Deletion and Pairwise Deletion. The Imputation branch splits into five subtypes: Single Imputation (e.g., mean, median, mode, LOCF-Last Observation Carried Forward, NOCB-Next Observation Carried Backward), Multiple Imputation, Model-Based Imputation (e.g., regression, k-nearest neighbor, hot-deck, maximum likelihood, expectation maximization), Machine Learning Imputation (e.g., artificial neural networks, deep learning), and Optimization Algorithm Imputation (e.g., genetic algorithm, particle swarm optimization, matrix completion). The Built-in branch includes Decision Tree Methods.

Reviewer: ```suggestion
alt: Flowchart about Missing Data Handling Techniques, adapted from Joel et al., 2022. The diagram presents three main strategies: Deletion, Imputation, and Built-in. Under Deletion, two methods are listed: Listwise Deletion and Pairwise Deletion. The Imputation branch splits into five subtypes: Single Imputation (for example., mean, median, mode, LOCF-Last Observation Carried Forward, NOCB-Next Observation Carried Backward), Multiple Imputation, Model-Based Imputation (e.g., regression, k-nearest neighbor, hot-deck, maximum likelihood, expectation maximization), Machine Learning Imputation (e.g., artificial neural networks, deep learning), and Optimization Algorithm Imputation (e.g., genetic algorithm, particle swarm optimization, matrix completion). The Built-in branch includes Decision Tree Methods.
```

> alt: Flowchart about Missing Data Handling Techniques, adapted from Joel et al., 2022. The diagram presents three main strategies: Deletion, Imputation, and Built-in. Under Deletion, two methods are listed: Listwise Deletion and Pairwise Deletion. The Imputation branch splits into five subtypes: Single Imputation (for example., mean, median, mode, LOCF-Last Observation Carried Forward, NOCB-Next Observation Carried Backward), Multiple Imputation, Model-Based Imputation (e.g., regression, k-nearest neighbor, hot-deck, maximum likelihood, expectation maximization), Machine Learning Imputation (e.g., artificial neural networks, deep learning), and Optimization Algorithm Imputation (e.g., genetic algorithm, particle swarm optimization, matrix completion). The Built-in branch includes Decision Tree Methods.

Reviewer: ```suggestion
alt: Flowchart about Missing Data Handling Techniques, adapted from Joel et al., 2022. The diagram presents three main strategies: Deletion, Imputation, and Built-in. Under Deletion, two methods are listed: Listwise Deletion and Pairwise Deletion. The Imputation branch splits into five subtypes: Single Imputation (for example, mean, median, mode, LOCF-Last Observation Carried Forward, NOCB-Next Observation Carried Backward), Multiple Imputation, Model-Based Imputation (for example, regression, k-nearest neighbor, hot-deck, maximum likelihood, expectation maximization), Machine Learning Imputation (for example, artificial neural networks, deep learning), and Optimization Algorithm Imputation (e.g., genetic algorithm, particle swarm optimization, matrix completion). The Built-in branch includes Decision Tree Methods.
```

> alt: Flowchart about Missing Data Handling Techniques, adapted from Joel et al., 2022. The diagram presents three main strategies: Deletion, Imputation, and Built-in. Under Deletion, two methods are listed: Listwise Deletion and Pairwise Deletion. The Imputation branch splits into five subtypes: Single Imputation (for example, mean, median, mode, LOCF-Last Observation Carried Forward, NOCB-Next Observation Carried Backward), Multiple Imputation, Model-Based Imputation (for example, regression, k-nearest neighbor, hot-deck, maximum likelihood, expectation maximization), Machine Learning Imputation (for example, artificial neural networks, deep learning), and Optimization Algorithm Imputation (e.g., genetic algorithm, particle swarm optimization, matrix completion). The Built-in branch includes Decision Tree Methods.

Reviewer: ```suggestion
alt: Flowchart about Missing Data Handling Techniques, adapted from Joel and others, 2022. The diagram presents three main strategies: Deletion, Imputation, and Built-in. Under Deletion, two methods are listed: Listwise Deletion and Pairwise Deletion. The Imputation branch splits into five subtypes: Single Imputation (for example, mean, median, mode, LOCF-Last Observation Carried Forward, NOCB-Next Observation Carried Backward), Multiple Imputation, Model-Based Imputation (for example, regression, k-nearest neighbor, hot-deck, maximum likelihood, expectation maximization), Machine Learning Imputation (for example, artificial neural networks, deep learning), and Optimization Algorithm Imputation (for example, genetic algorithm, particle swarm optimization, matrix completion). The Built-in branch includes Decision Tree Methods.
```
